CONTRIBUTING.md

```diff
@@ -47,3 +48,53 @@ Improve contributing docs and consolidate them in the standard location https://
 ```
 
 ### For more information on Cumulus governance, see the [Cumulus Code Contribution Guidelines](https://docs.google.com/document/d/14J_DS6nyQ32BpeVjdR-YKfzHAzFB299tKghPGshXUTU/edit) and [the Cumulus Wiki](https://wiki.earthdata.nasa.gov/display/CUMULUS/Cumulus).
+
+## Getting started for beginners 
```

Comment:
To keep with the current heading style consistency: All words are title case except for prepositions and coordinating conjunctions. Any reference to this section heading needs to be addressed as well (above for the link).